db scripts/Contact_functions/update_exposures_function.sql

```diff
@@ -0,0 +1,86 @@
+CREATE OR REPLACE FUNCTION public.update_exposures_function(input_exposures json)
```

Comment:
Same here

Reply:
Daniel's function is more updated then this, due to another pr that changed it